/* 
image_library.proto: Protobuf representation of an image library for the scoreboard.

Copyright 2020-2023 Tracy Beck

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/

syntax = "proto3";
package cszb_scoreboard.proto;

message ImageInfo {
	string name = 1;
	string file_path = 2;
	repeated string tags = 3;
	bool is_relative = 4;
}

message ImageLibrary {
	repeated ImageInfo images = 1;
	string library_root = 2;
}
